README.md

```diff
@@ -92,6 +92,7 @@ You have a third option : providing a function that returns the component's chil
 Features
 ========
 
+- `allowGlobalDrops` `[Boolean | **false**]` — Dropping files outside of the `<Dropzone>` is allowed. This can cause the current browser window to be taken over by the dropped file.
```

Comment:
Should this be renamed to something like `preventDropOnDocument` with the default value of `true`? I think right the purpose of it is not clear enough.